tutorials/getting_started/another_model_example.md

```diff
@@ -0,0 +1,611 @@
+that get passed to your `Model`.  A `TextField` represents a tokenized piece of text, a
+`LabelField` represents a categorical label, and there are other `Fields` representing other kinds
+of data that we don't need to use here.  These inputs are plain strings, but they will be converted
+to integer arrays before being passed to your `Model` after building a vocabulary from the training
+data.  All of this is handled for you by AllenNLP, in a way that you can configure.
+
+The second important thing to notice is that we're using some objects on `self` to configure how
+the `TextFields` are constructed: `self._tokenizer` and `self._token_indexers`.  The `Tokenizer`
+determines how the title and abstract get converted into tokens.  The `Tokenizer` could split the
+string into words, characters, byte pairs, or any other way you want to tokenize your data.  The
+`TokenIndexers` then determine how to represent these tokens as arrays in your model.  If your
+tokens are words, for instance, the `TokenIndexers` could make arrays out of the word id, the
+characters in the word, the word's part of speech, or many other things.  Notice that we're not
+specifying _any_ of that in the code, we're just saying that we'll get a `Tokenizer` and some
+`TokenIndexers` as input to this `DatasetReader`, and we'll use them to process our data.  This
+lets us be _really_ flexible later in how exactly our inputs get represented.  We can experiment
+with using character-level CNNs or POS-tag embeddings without changing our `DatasetReader` or our
+`Model` code at all.
+
+How does this work?  This brings us to the last pieces of our `DatasetReader`: the constructor and
+the `from_params` method.
+
+```python
+@DatasetReader.register("s2_papers")
+class SemanticScholarDatasetReader(DatasetReader):
+    def __init__(self,
+                 tokenizer: Tokenizer = None,
+                 token_indexers: Dict[str, TokenIndexer] = None) -> None:
+        self._tokenizer = tokenizer or WordTokenizer()
+        self._token_indexers = token_indexers or {"tokens": SingleIdTokenIndexer()}
+
+    @classmethod
+    def from_params(cls, params: Params) -> 'SemanticScholarDatasetReader':
+        tokenizer = Tokenizer.from_params(params.pop('tokenizer', {}))
+        token_indexers = TokenIndexer.dict_from_params(params.pop('token_indexers', {}))
+        params.assert_empty(cls.__name__)
+        return cls(tokenizer=tokenizer, token_indexers=token_indexers)
+```
+
+The constructor takes these dependencies as inputs, where by default we're splitting text into
+words, and representing them as single word ids (as opposed to, say, sequences of character ids).
+To get different tokenization behavior or different word representations, you just need to pass in
+different objects when you construct the `DatasetReader`.
+
+But where do I construct the `DatasetReader`, you ask?  Well, you don't, at least not if you're
```

Comment:
This rhetorical style is used quite a lot - it's a bit much for a tutorial, maybe.